src/nl/blog/2025/01/omscholen-naar-front-end-developer.md

```diff
@@ -0,0 +1,57 @@
+
+De bedrijven die oprecht enthousiast waren om mensen op te leiden en daar serieus in waren, wezen mij helaas af. De feedback was dat ze mensen zochten met meer ervaring. Dit voelde best naar, maar ik was alsnog vastberaden en dacht: ik regel het zelf wel.
+
+Ik ben bij Novi Hogeschool een omscholing gaan doen, ook al keek ik wel op tegen 7 maanden fulltime studeren. Achteraf ben ik er alleen maar blij mee. Zeven maanden voelde aan het begin heel lang, maar waren zo voorbij. Het was juist fijn om de tijd te hebben om je te ontwikkelen.
+
+Inmiddels zijn er veel meer bedrijven die goede traineeships aanbieden. Voor mij gaf de studie me wat meer comfort om zonder bedrijfsdruk mezelf te ontwikkelen. Vraag vooral goed door bij een bedrijf: je komt er vanzelf achter of het een match is van beide kanten of dat je toch liever voor een losse studie kiest.
+
+### Hoe vind je je eerste baan als front-end developer?
+Je hebt inmiddels bloed, zweet en tranen in je omscholing gestopt. Geloof me, het is echt bikkelen. Je moet het écht willen, en zelfs dan is het zwaar. Maar tegelijkertijd is het ook zo de moeite waard!
+
+Nu ben je toe aan de volgende stap: een heuse baan! Tijdens mijn studie maakte ik me daar best wel zorgen over. Ik kon me niet voorstellen dat een bedrijf iemand zou aannemen met zo weinig front-end werkervaring als ik.
+
+Dit ging echter veel gemakkelijker dan ik dacht. Nou ja, soort van gemakkelijk. Ik had een oproep op LinkedIn geplaatst waarin ik vertelde dat ik een toffe baan als front-ender zocht, waarom ik een omscholing had gedaan, en wat voor type bedrijf ik zocht. Binnen no-time kreeg ik behoorlijk wat reacties. Uiteindelijk had ik 8 aanbiedingen!
+
+Dit overweldigde me enorm, ik had dit niet verwacht. Dus hier twee tips:
+
+1. Een oproep op LinkedIn is heel lucratief
+2. Wees zelf ook selectief waar je solliciteert
+
+Ga niet, zoals ik, overal solliciteren. Dat maakt het alleen maar moeilijker om een keuze te maken. Vanuit verschillende bedrijven werd ook behoorlijk wat druk op mij gezet om snel een keuze te maken. Achteraf gezien zou ik hier meer tijd voor hebben willen nemen. Bedrijven willen je graag, dus als jij meer tijd nodig hebt, zeg dat gewoon en pak die tijd.
+
+### Conclusie: just do it
+
+Na alle tips, adviezen en ervaringen, een laatste raad: voelt het goed? Ga er gewoon voor. De IT biedt enorm veel mooie kansen. Er zijn genoeg toffe banen, veel doorgroeimogelijkheden, en je bent oprecht een waardevolle aanvulling als omscholer. Juist omdat je meer meeneemt dan iemand zonder eerdere werkervaring.
+
+Heel veel plezier en succes met je nieuwe carrière!
+
+_Handige bronnen:_
```

Comment:
Eventueel kun je hier wel een h2 of h3 van maken

```suggestion
## Handige bronnen:
```

Reply:
Thanks! Aangepast in 253bca2ecea5d0495da88c1e8d223f7b34356dd4